src/alert_manager.cpp

```diff
@@ -52,8 +52,7 @@ namespace libtorrent
 
 	int alert_manager::num_queued_resume() const
 	{
-		mutex::scoped_lock lock(m_mutex);
-		return m_num_queued_resume;
+		return m_num_queued_resume.load(boost::memory_order_consume);
```

Comment:
I think `consume` here is clearly wrong. I think anything other than "sequential consistency" requires very strong arguments.